content/ui-patterns/degraded-experiences.mdx

```diff
@@ -54,80 +54,18 @@ In addition to a global banner, we should inform users about availability issues
 
 ## Degraded page content
 
-If part of the UI cannot be rendered or would be rendered without critical information, default to not rendering it at all.
-
 Before making a decision about how to handle UI with unavailable content, check if it already has guidelines on handling errors or empty states.
 
-### Removing UI
-
-You can remove the affected UI if it's not critical to core workflows and it wouldn't be confusing to render the rest of the page without that UI. For example: it's ok to hide the reactions button.
-
-Examples of UI that might be disorienting to remove:
-
-- The comment box on issues and pull requests
-- The "Request changes" button in the pull request "Changes" tab
-- Submit buttons on forms
-
-<Box
-  mb={3}
-  display="flex"
-  alignItems="flex-start"
-  justifyContent="space-between"
-  flexDirection={['column', 'column', 'column', 'column', 'row']}
-  sx={{gap: 3}}
->
-
-<div>
-
-#### Handling unavailable counts
-
-When the data required to calculate a count is unavailable, default to hiding the number. If the count is shown inside of an interactive element, a tooltip may be displayed on focus and hover to explain the missing count.
-
-</div>
-
-<img
-  width="456"
-  alt="Two images of repo tabs: 1. Tabs have counts; 2. Tabs don't have counts;"
-  src="https://github.com/primer/design/assets/2313998/0f368a1d-78fa-4ec4-92ab-7f587fb93a7e"
-/>
-
-</Box>
-
-<Box
-  mb={3}
-  display="flex"
-  alignItems="flex-start"
-  justifyContent="space-between"
-  flexDirection={['column', 'column', 'column', 'column', 'row']}
-  sx={{gap: 3}}
->
-
-<div>
-
-#### Handling activity indicators
-
-When the data is unavailable to determine whether to show an activity indicator (most commonly used for notification badges), default to hiding the indicator.
-
-</div>
-
-<img
-  width="456"
-  alt="Image of the notification link button in the global nav without a badge"
-  src="https://github.com/primer/design/assets/2313998/4f39547c-749b-4e44-a15f-05d18a1dc0c1"
-/>
-
-</Box>
-
 ### Replacing UI with a message
 
+We don't want users to think they've suffered data loss. If we know a user created something, don't show a generic "empty state". It's better to explain that it's unavailable, or remove the entire section from the page (including the section's heading). For example, I might think my repositories were deleted if I'm on an organization page (like [https://github.com/primer/](https://github.com/primer/)) and the repositories aren't listed there.
```

Comment:
```suggestion
We don't want users to think they've suffered data loss. Instead of a generic "empty state, it is better to explain that it's unavailable, or remove the entire section from the page (including the section's heading). For example, as a user, I might think my repositories were deleted if I am on an organization page (like [https://github.com/primer/](https://github.com/primer/)) and the repositories aren't listed there.
```